components/kserve/resources/servicemesh/routing/kserve-local-gateway-svc.tmpl.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    experimental.istio.io/disable-gateway-port-translation: "true"
```

Comment:
To be honest, I don't know about this label meaning in detail. I just tried to use the same label that knative-local-gateway used. I am going to update this label to what you referred.

Reply:
Just followed the knative-local-gateway. Could you please guide me to change the part to long term solution?

Reply:
Let me clarify. I wanted to emphasize the importance of capturing this information, as it appears to be "experimental" and may be deprecated or removed in the future. Since a long-term solution is available, we should consider transitioning to it. However, perhaps we should address it with upstream KServe first?

Reply:
OK, let's research this with another ticket. For now, we can leave it.

Reply:
@Jooho can you link it here?